projects/addon-commerce/tokens/index.ts

```diff
@@ -1 +1,2 @@
 export * from './i18n';
+export * from './card-cvc';
```

Comment:
Oops, sorry, let's move it to i18n like the rest. I thought you meant i18n package.